tracer/contrib/elastictraced/elastictraced.go

```diff
@@ -0,0 +1,98 @@
+// Package elastictraced provides tracing for the Elastic Elasticsearch client (https://github.com/olivere/elastic)
```

Comment:
Are we compatible only with `v5`? 
We should explicit the compatible versions in the doc (and if that's more than `v5`, we should also add the others in the tests).